strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BrowseController.java

```diff
@@ -63,13 +71,11 @@
     public ResponseEntity<String> browse()
     {
         try
-        {
-            Map<String,SortedSet<String>> storages = new HashMap<>();
-            storages.put("storages", new TreeSet<String>(configurationManager.getConfiguration()
-                    .getStorages()
-                    .keySet()));
+        {   
+            Path dirPath = getStoragesDir();
```

Comment:
In the original implementation storages was fetched from configuration. 
@sanket407 have you approved with @carlspring that storages will be treated as folders now?

Reply:
@sbespalov I had implemented them similarly in BrowseStoragesController. There it was approved. @carlspring What do you suggest ?

Reply:
I believe @sbespalov has a point here. The thing is this: sure, one would normally think that clicking ".." should take you one level up, but then again... what happens if you have some sensitive directories which you've put there, (as an admin), directly on the file-system? For example, you've disabled (commented out the configuration for this in the `strongbox.xml`) some storage for the moment, or have some back up there on the file system... Since it's disabled, people shouldn't be able to browse it.
But... then you would have to the the same for repositories as well...

So, in conclusion, I believe @sbespalov is right here and we should do it as it was (and also apply this change to repositories).